tmail-backend/blob/blobid-list/pom.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>tmail-backend</artifactId>
+        <groupId>com.linagora.tmail</groupId>
+        <version>1.0.0-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>blobid-list</artifactId>
+    <name>Blob :: BlobIdList</name>
+    <description>Single save blob store</description>
```

Comment:
```suggestion
    <description>When deduplication is active, use a Cassandra projection to avoid saving several time the same blob to S3.
    This prevents isolation issues for Swift backed S3 deployments and also  adds a performance boost.</description>
```